scanamo/src/main/scala/org/scanamo/generic/AutoDerivation.scala

```diff
@@ -16,17 +16,12 @@
 
 package org.scanamo.generic
 
-import magnolia.Magnolia
-import org.scanamo.DynamoFormat
+/**
+  * This class enables the automatic derivation of a [[org.scanamo.DynamoFormat]] instance if in lexical scope.
+  * The actual derivation happens inside the companion
+  */
+private[scanamo] class AutoDerivationUnlocker
```

Comment:
```suggestion
private[scanamo] sealed abstract class AutoDerivationUnlocker
```

Reply:
Done in ee1f8e53186959626425e38ab4bca4d179d60ff6